src/lib/core/api/DataClient/types.ts

```diff
@@ -237,22 +227,43 @@ export const ScatterplotResponseData = array(
   })
 );
 
+// typing computedVariableMetadata for computation apps such as alphadiv and abundance
+export type ComputedVariableMetadata = TypeOf<typeof ComputedVariableMetadata>;
+export const ComputedVariableMetadata = partial({
+  displayName: array(string),
+  displayRangeMin: string,
+  displayRangeMax: string,
+  // collectionVariable for abundance app
+  collectionVariable: partial({
+    collectionType: string,
+    collectionValuePlotRef: string, // e.g., yAxisVariable
```

Comment:
Just thinking for the future, the collectionValuePlotRef and collectionVariablePlotRef will give the information of where collection var data will go. In other words, it is a way for us to get information about what parts of the plot the app will override/supply data for.

Reply:
@asizemore yes I also guessed about that when I checked actual(!) backend response and the Ref values 👍: that's the reason why I could add e.g. texts in there haha 😄 That info may be helpful to fill out VariableCoverageTable's X, Y, overlay variables' labels, I suppose